### DISCUSSION TITLE
Cookie notice?

### DISCUSSION THREAD

**Discussion by JamesMcGhee** [GitbookIO/community#764] [Feature Requests]
Is there no cookie notice on the resulting books, that would present a problem with compliance in several places seems like an easy thing to apply

**Comment by SamyPesse** [comment]
Hi @JamesMcGhee, we do have a cookie banner being displayed. It's only displayed when needed: if you use an integration that requires cookies (ex: Google Analytics, etc) or if you configure a privacy policy URL in your site customization.
